src/importexport/imagesexport/internal/pdfwriter.cpp

```diff
@@ -65,9 +65,15 @@ Ret PdfWriter::write(INotationPtr notation, io::IODevice& destinationDevice, con
         return false;
     }
 
+    //transparency handling
```

Comment:
This comment doesn't add much information either; we prefer only writing comments when they contain information that cannot (easily) be deduced from the code itself.